<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bFactor - Local Resolution</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/prefix1/icons/xmipp_logo.png</normaloff>:/new/prefix1/icons/xmipp_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>60</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(103, 175, 106);
background-color: rgb(0, 150, 128);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="buttonXmipp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/new/prefix1/icons/xmipp_logo.png</normaloff>:/new/prefix1/icons/xmipp_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>33</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="linePath">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>18</y>
         <width>401</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonPath">
       <property name="geometry">
        <rect>
         <x>515</x>
         <y>18</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/new/prefix1/icons/path.png</normaloff>:/new/prefix1/icons/path.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonHelp">
       <property name="geometry">
        <rect>
         <x>545</x>
         <y>18</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/new/prefix1/icons/helpblue.png</normaloff>:/new/prefix1/icons/helpblue.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonCite">
       <property name="geometry">
        <rect>
         <x>575</x>
         <y>18</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/new/prefix1/icons/cite.png</normaloff>:/new/prefix1/icons/cite.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabParams">
         <attribute name="title">
          <string>Parameters</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="labelAtModel">
              <property name="text">
               <string>Atomic Model (pdb)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Normalize Resolution</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelLocRes">
              <property name="text">
               <string>Local Resolution Map</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineSampling">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="ButtonAtModel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/prefix1/icons/open.png</normaloff>:/new/prefix1/icons/open.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="ButtonViewLocRes">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/prefix1/icons/see.png</normaloff>:/new/prefix1/icons/see.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="ButtonLocRes">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/prefix1/icons/open.png</normaloff>:/new/prefix1/icons/open.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineLocRes"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="linePDB"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelSampling">
              <property name="text">
               <string>Sampling Rate (A)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkNormalize">
                <property name="text">
                 <string>Yes</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelFSCNormalize">
                <property name="text">
                 <string>FSC Resolution (A)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineFSC">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabAdvance">
         <attribute name="title">
          <string>Advance params</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Use median</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item alignment="Qt::AlignLeft">
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>Yes</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QRadioButton" name="checkMedianNo">
                <property name="text">
                 <string>No</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="executeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="resultsButton">
        <property name="text">
         <string>Results</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="downFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 150, 128);
sdfzsfzsfsf</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="labelMonoRes">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Matching Local Resolution and bfactor</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelCiteMonoRes">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Author et. al. Journal (2021)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
